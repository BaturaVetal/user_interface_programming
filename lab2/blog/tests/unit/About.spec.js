import { shallowMount } from '@vue/test-utils';
import AboutComponent from '@/components/About.vue';


describe('About.vue', () => {
  it('renders about page content', () => {
    const wrapper = shallowMount(AboutComponent);
    expect(wrapper.text()).toContain('Цей додаток є блогом');
  });


  it('matches the snapshot', () => {
    const wrapper = shallowMount(AboutComponent);
    expect(wrapper.element).toMatchSnapshot();
  });
});
