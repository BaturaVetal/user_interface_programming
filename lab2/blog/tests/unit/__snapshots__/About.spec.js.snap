// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue matches the snapshot 1`] = `
<v-card>
  <v-card-title>
    Про додаток
  </v-card-title>
  <v-card-text>
    <p>
      Цей додаток є блогом з можливістю додавання, перегляду та коментування повідомлень. Розробив студент ФПМ КВ-31мн Батура Віталій.
    </p>
  </v-card-text>
</v-card>
`;
